packages/experimental/Menu/src/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,45 @@
+/** @jsx withSlots */
+import { View } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+import { compose, mergeProps, UseSlots, withSlots } from '@fluentui-react-native/framework';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { menuItemName, MenuItemProps, MenuItemType } from './MenuItem.types';
+import { useMenuItem } from './useMenuItem';
+
+export const MenuItem = compose<MenuItemType>({
+  displayName: menuItemName,
+  tokens: [],
+  slotProps: {
+    root: {
+      style: { height: 48, width: 200, flexDirection: 'row', alignItems: 'center', backgroundColor: 'transparent' },
+    },
+    content: {
+      color: 'white',
+    },
+  },
+  slots: {
+    root: View,
+    content: Text,
+    submenuIndicator: SvgXml,
+  },
+  useRender: (userProps: MenuItemProps, useSlots: UseSlots<MenuItemType>) => {
+    const menuItem = useMenuItem(userProps);
+    // grab the styled slots
+    const Slots = useSlots(userProps);
+
+    return (final: MenuItemProps) => {
+      const mergedProps = mergeProps(menuItem.props, final);
+      const chevronXml = `
+      <svg width="12" height="16" viewBox="0 0 11 6">
```

Comment:
Yeah I just stole this from somewhere else and I don't think it's the final one since it points down instead of left/right. So we'll replace it eventually.

Reply:
Looking at https://github.com/microsoft/fluentui-system-icons/blob/master/assets/Chevron%20Right/SVG/ic_fluent_chevron_right_16_regular.svg, and the raw text of the SVG, it doesn't look like it's actually simpler 😅
```
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="[http://www.w3.org/2000/svg">](http://www.w3.org/2000/svg%22%3E)
<path d="M5.64645 3.14645C5.45118 3.34171 5.45118 3.65829 5.64645 3.85355L9.79289 8L5.64645 12.1464C5.45118 12.3417 5.45118 12.6583 5.64645 12.8536C5.84171 13.0488 6.15829 13.0488 6.35355 12.8536L10.8536 8.35355C11.0488 8.15829 11.0488 7.84171 10.8536 7.64645L6.35355 3.14645C6.15829 2.95118 5.84171 2.95118 5.64645 3.14645Z" fill="#212121"/>
</svg>
```

Reply:
Ohhh I forgot that all the icons were in a repo now. This is nice.

Eh well, not much we can do about it 😅

Reply:
Yeah, oh well 🤷🏾. There _is_ a way to take an npm dependency on that repo and get back a React function component representing the icon that we never explored. I'm happy with leaving this as is since that would probably mean we have to bundle assets or something